os/contiki-default-conf.h

```diff
@@ -58,6 +58,16 @@
 #ifndef QUEUEBUF_CONF_NUM
 #define QUEUEBUF_CONF_NUM 8
 #endif /* QUEUEBUF_CONF_NUM */
+
+/* RINGBUF_INDEX_CONF_TYPE defines the default type of ringbuf index. 
+   access to (struct ringbuf *)->get_ptr and ->get_ptr must be atomic. 
+   users cans change to the type which makes access atomically, 
+   but C does not guarantee this.
+ */
+#ifndef RINGBUF_INDEX_CONF_TYPE
+#define RINGBUF_INDEX_CONF_TYPE  uint8_t
```

Comment:
Because I'm used to using the default ringbuf library, so i make this commit.
this is just a suggestion, the specific changes are still up to you. But I still sincerely hope that you can change it, so I don't need to manually change it every time.
Now I will close this PR.
Thanks!